phases/libs/qualityRubric.json

```diff
@@ -2,163 +2,50 @@
   "categories": [
     {
       "name": "Naming",
-      "value": 7.5,
-      "items": [
-        {
-          "reporters": [
-            {
-              "name": "[PackageName]",
-              "weight": 1
-            }
-          ],
-          "weight": 0
-        },
-        {
-          "reporters": [
-            {
-              "name": "[TypeName]",
-              "weight": 1
-            }
-          ],
-          "weight": 3
-        },
-        {
-          "reporters": [
-            {
-              "name": "[MethodName]",
-              "weight": 1
-            }
-          ],
-          "weight": 3
-        },
-        {
-          "reporters": [
-            {
-              "name": "[ConstantName]",
-              "weight": 0
-            },
-            {
-              "name": "[MemberName]",
-              "weight": 3
-            },
-            {
-              "name": "[LocalVariableName]",
-              "weight": 3
-            },
-            {
-              "name": "[RecordComponentName]",
-              "weight": 3
-            },
-            {
-              "name": "[ParameterName]",
-              "weight": 3
-            },
-            {
-              "name": "[LambdaParameterName]",
-              "weight": 2
-            },
-            {
-              "name": "[StaticVariableName]",
-              "weight": 2
-            },
-            {
-              "name": "[LocalFinalVariableName]",
-              "weight": 1
-            },
-            {
-              "name": "[PatternVariableName]",
-              "weight": 1
-            }
-          ],
-          "weight": 5
-        },
-        {
-          "reporters": [
-            {
-              "name": "[RecordTypeParameterName]",
-              "weight": 1
-            },
-            {
-              "name": "[MethodTypeParameterName]",
-              "weight": 1
-            },
-            {
-              "name": "[ClassTypeParameterName]",
-              "weight": 1
-            },
-            {
-              "name": "[InterfaceTypeParameterName]",
-              "weight": 1
-            }
-          ],
-          "weight": 1
-        }
+      "value": 25,
+      "reporters": [
+        "[PackageName]",
```

Comment:
I really like the grouping of multiple `reporters` into a single `category`. What is the purpose of the square brackets around each of the names? Is that part of the string matching, or part of the official name of the rule? If it's not _part of the name_, could we remove them and make it part of the processing phase?

Reply:
That is how it's spit out by checkstyle, but I'll try to put it into the processing instead

Reply:
How's [this?](https://github.com/softwareconstruction240/autograder/pull/207/commits/06e599fd57778adac33c951f6bbdf0d1164dc8cf)

Reply:
I love it! Thanks 😄